packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -32,17 +32,32 @@ export const FinalFormSelect = <T,>({
     children,
     endAdornment,
     clearable,
+    multiple,
     ...rest
 }: FinalFormSelectProps<T> & Partial<AsyncOptionsProps<T>> & Omit<SelectProps, "input">) => {
     const selectEndAdornment = clearable ? (
-        <ClearInputAdornment position="end" hasClearableContent={Boolean(value)} onClick={() => onChange(undefined)} />
+        <ClearInputAdornment
+            position="end"
+            hasClearableContent={Boolean(multiple ? (Array.isArray(value) ? value.length : value) : value)}
```

Comment:
According to the description of MUI Select's `multiple` prop, the select requires an array, and we basically just wrap that, so no.

>  If `true`, `value` must be an array ...

https://mui.com/material-ui/api/select/#props

Reply:
I believe you can remove the Array.isArray check here as you already check for `multiple`

Reply:
I need this so TS knows it can access `.length`. 
Do you think I should add something like this to the props? Seems a bit overkill to me 🤷🏼 

```ts
type MultipleProps = {
    multiple: true;
    value: Array<Val>;
}

type NonMultipleProps = {
    multiple?: false;
    value: Val;
}

type Props = { ... } & (MultipleProps | NonMultipleProps)
```

Reply:
No, let's leave the Array.isArray check 🙂